virt/virtual_machines/virt-managing-vms-openshift-pipelines.adoc

```diff
@@ -0,0 +1,51 @@
+:_content-type: ASSEMBLY
+[id="virt-managing-vms-openshift-pipelines"]
+= Managing virtual machines with OpenShift Pipelines
+include::_attributes/common-attributes.adoc[]
+:context: virt-managing-vms-openshift-pipelines
+
+toc::[]
+
+xref:../../cicd/pipelines/understanding-openshift-pipelines.adoc#understanding-openshift-pipelines[Red Hat OpenShift Pipelines] is a Kubernetes-native CI/CD framework that allows developers to design and run each step of the CI/CD pipeline in its own container.
+
+The Tekton Tasks Operator (TTO) integrates {VirtProductName} with OpenShift Pipelines. TTO includes cluster tasks and example pipelines that allow you to:
+
+* Create and manage virtual machines (VMs), persistent volume claims (PVCs), and data volumes
+* Execute commands in VMs
```

Comment:
Another cautionary tale from the ISG; there are also a few other instances of "execute" in this PR that you might want to look at.

Use with caution 
execute   verb
Use only if a simpler term, such as "run", "issue", "start", or "enter" is inappropriate.

Reply:
Replaced all instances of "execute" with "run"